src/khoj/routers/twilio.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+
+from twilio.rest import Client
+
+from khoj.database.models import KhojUser
+
+logger = logging.getLogger(__name__)
+
+account_sid = os.getenv("TWILIO_ACCOUNT_SID")
+auth_token = os.getenv("TWILIO_AUTH_TOKEN")
+verification_sid = os.getenv("TWILIO_VERIFICATION_SID")
+
+twilio_enabled = account_sid and auth_token and verification_sid
+if twilio_enabled:
+    client = Client(account_sid, auth_token)
+
+
+def is_twilio_enabled():
+    return twilio_enabled
+
+
+def create_otp(user: KhojUser):
+    """Create a new OTP for the user"""
+    verification = client.verify.v2.services(verification_sid).verifications.create(
+        to=str(user.phone_number), channel="whatsapp"
+    )
+    return verification.sid
```

Comment:
what is this `verification.sid` meant to be used for? Names are a bit confusing between `verification`, `verification.sid` and the package level `verification_sid` variables